app/stagemanager.js

```diff
@@ -114,9 +123,10 @@ class StageManager extends MyTrello {
 
     closeList(listData, trelloListID) {
         var deferred = Q.defer();
-        if (!listData.length || listData.length) {
+        console.log(listData);
+        if (!listData.length || listData.length == 0) {
```

Comment:
Similar to the above comment, if this conditional fails, the promise never resolves or rejects.